app/Ptr/PtrControllerTest.php

```diff
@@ -156,6 +158,21 @@ public function testListUpdatesOnCreate() {
     });
   }
 
+  public function testInvalidPtrIpOnCreate(){
+    $this->asAdminWithPermissions(static::PERMISSIONS, function () {
+      $this->dns
+      ->shouldReceive('get')
+      ->with('test_create')
+      ->once()
+      ->andReturn(
+        array(
+          array('host' => 'test_create','ip'=> '1.1.1.3', 'type' => 'A')
+        ));      $ip = "9.9.9.9";
+      $this->tryCreate(['ip' => $ip]);
+      $this->assertResponseStatus(409);
```

Comment:
can we assert that we also see a certain error message that is user-facing and actionable?